CONTRIBUTING.md

```diff
@@ -0,0 +1,114 @@
+# Contributing 
+This is a document outlining how to contribute to the Tūhura Tech wiki.
+
+There are two main ways of contributing:  
+- Suggesting - this involves creating a new [issue on Github](https://github.com/Tuhura-Tech/Wiki/issues) and informing us of a way to improve the wiki. Alturnatively, you can notify the team directly on our [Discord server](https://github.com/Tuhura-Tech/Wiki/issues)
```

Comment:
We should run spellcheck and also validate some of the links such as [Discord Server]

Reply:
Yeah I just ran it through google docs and proofread it now, I'll do that sooner in the future